### PR TITLE
fix: manifest v3

### PR DESCRIPTION
**Pull request by Dudleyneedham**
Fixing the manifest file as per Toms old PR and branch

### CODE REVIEW

File: src/static/manifest.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "__MSG_manifest_name__",
   "short_name": "__MSG_manifest_short_name__",
   "description": "__MSG_manifest_description__",
-  "version": "2024.9.11",
+  "version": "2024.2.19",
```
Comment (kilted-andres) [inline src/static/manifest.json:6]: This needs to be updated.

**Review by kilted-andres — CHANGES_REQUESTED** [review]
Maybe its good to test the whole extension before pushing this. Just from looking its hard to tell if it works. 

The version needs to be updated.

File: src/static/manifest.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "__MSG_manifest_name__",
   "short_name": "__MSG_manifest_short_name__",
   "description": "__MSG_manifest_description__",
-  "version": "2024.9.11",
+  "version": "2024.9.23",
```
Comment (arty-name) [inline src/static/manifest.json:6]: Could you do a global search-and-replace of 2024.9.11 to 2024.9.23? Tests will fail otherwise.

Comment (Dudleyneedham) [reply src/static/manifest.json:6]: I thought it was just one place. THis should now be updated.

**Review by kilted-andres — APPROVED** [review]
I build, packaged and smoke-tested this Sporran version locally.

**Pull request merged**